### ISSUE TITLE
Improving - "What are the issues?" section

### ISSUE THREAD

**Issue by sharronrush** [w3c/wai-planning-and-implementation#102]
1. Intro sentence too wordy, suggested change to:

Skip this section if you have already identified accessibility issues. However, if you want to gain basic orientation to possible issues and get a rough idea of the current state of accessibility, consider these resources:
1. Omit the sentence:  Some web accessibility evaluation tools may help you further explore the accessibility on your website:  since the short description phrase explains its purpose.
2. Omit:  Note: some web accessibility evaluation tools may be complex and confusing to use for people new to accessibility. They typically only check a subset of issues, and may be prone to misleading results.
   Rationale: This is too much detail for a "first aid" guide. The first paragraph of the Tools List has support for choosing a tool that includes that caution.

**Comment by susanatx** [comment]
1. Further edit

Skip this section if you are already aware of your accessibility issues. But, if you need to identify  potential issues, consider these resources:

Rationale: Even less wordy and I don't think of the links here address "the current state of accessibility." I'm not sure that's even necessary for someone at this stage.

Omitted sentences...I agree about ditching the warning sentence as it begs the question of why we're pointing to this person in panic mode to these if they're complex, confusing, and possibly misleading. The only warning on the tools list I see though is the disclaimer that W3C doesn't endorse them. Is that what you mean @sharronrush?

**Comment by nitedog** [comment]
1. Changed intro sentences following these suggestions.
2. Removed warnings and merged "easy checks" and "tools list" into one list
3. Added h3 "Detailed Review" to separate WCAG-EM from the rest
